pkg/scalers/rabbitmq_scaler.go

```diff
@@ -61,18 +62,19 @@ type rabbitMQScaler struct {
 }
 
 type rabbitMQMetadata struct {
-	queueName   string
-	mode        string        // QueueLength or MessageRate
-	value       int           // trigger value (queue length or publish/sec. rate)
-	host        string        // connection string for either HTTP or AMQP protocol
-	protocol    string        // either http or amqp protocol
-	vhostName   *string       // override the vhost from the connection info
-	useRegex    bool          // specify if the queueName contains a rexeg
-	pageSize    int64         // specify the page size if useRegex is enabled
-	operation   string        // specify the operation to apply in case of multiples queues
-	metricName  string        // custom metric name for trigger
-	timeout     time.Duration // custom http timeout for a specific trigger
-	scalerIndex int           // scaler index
+	queueName      string
+	mode           string        // QueueLength or MessageRate
+	value          int           // trigger value (queue length or publish/sec. rate)
+	host           string        // connection string for either HTTP or AMQP protocol
+	protocol       string        // either http or amqp protocol
+	vhostName      *string       // override the vhost from the connection info
+	useRegex       bool          // specify if the queueName contains a rexeg
+	pageSize       int64         // specify the page size if useRegex is enabled
+	operation      string        // specify the operation to apply in case of multiples queues
+	metricName     string        // custom metric name for trigger
+	timeout        time.Duration // custom http timeout for a specific trigger
+	scalerIndex    int           // scaler index
+	minMetricValue float64       // min metric value
```

Comment:
please, use `int64`. The HPA doesn't use `float64` so we are trying to avoid unneeded casts

Reply:
In this case, I think could make sense. Since there are two options for `value`. Messages queued or publish rate. Since publish rate is float could make sense for this.

Reply:
you are right, but even that, the HPA only uses int64, so we have to cast the value into int64 when we create the metrics